src/main/java/com/uid2/shared/secure/AzureCCCoreAttestationService.java

```diff
@@ -18,19 +19,21 @@
 // CC stands for Confidential Container
 @Slf4j
 public class AzureCCCoreAttestationService implements ICoreAttestationService {
+    private final String attestationUrl;
```

Comment:
This is not used in this class. It is only passed to the constructor of the PolicyValidator